controllers/managedocs_controller.go

```diff
@@ -21,6 +21,7 @@ import (
 	"encoding/json"
 	"fmt"
 	"io/ioutil"
+	"k8s.io/apimachinery/pkg/api/resource"
```

Comment:
the import order is not proper. this should we are mixing core packages and the third part packages